sum_of_nth_terms.py

```diff
@@ -0,0 +1,16 @@
+'''My code for sum-of-nth-terms kata.'''
+
+
+def series_sum(n):
+    '''Add 1 divided by a div increasing by three for range n.'''
+    if n == 0:
+        return '0.00'
+    acc = 0
+    div = -2
+    for i in range(n):
+        div += 3
+        acc += 1 / div
```

Comment:
Missed this the first time I went through but something seemed off to me. Sorry for not being thorough earlier. You're doing integer division, and in Python 2 that means your division won't work the way you want it to. All you get is "1.00" if n > 0